lib/src/base/paged_layout_builder.dart

```diff
@@ -162,7 +162,7 @@ class _PagedLayoutBuilderState<PageKeyType, ItemType>
   @override
   void initState() {
     super.initState();
-    if (_state.status == PagingStatus.loadingFirstPage) {
+    if (_state.status == PagingStatus.loadingFirstPage && !_state.isLoading) {
```

Comment:
This add a little more flexibility : if initial state is already loading, why start a new one.